src/docs/sphinx/ARM.rst

```diff
@@ -4,6 +4,70 @@
 .. # SPDX-License-Identifier: MIT
 
 
-ARM Overview (under development)
+Arm Overview
 ================================
 
+This implementation of the Arm port of Variorum supports the Arm Juno r2 SoC.
+The Arm Juno r2 platform is a big.LITTLE cluster with Cortex-A72 (big) and
+Cortex-A53 (LITTLE) clusters (also called processors), respectively.
+It also has an Arm Mali GP-GPU. We have tested the Arm functionality of
+Variorum on Linaro supported Linux.
+
+References:
+
+* `Arm Juno r2 technical reference <https://developer.arm.com/documentation/100114/0200/>`_
+
+Monitoring and Control Through Sysfs Interface
+-----------------------------------------------
+
+The built-in monitoring interface on the Arm Juno r2 board is implemented
+by the on-board FPGA. Since this interface is not universally available
+on most Arm implementations, we leverage the standard Linux sysfs interface
+for monitoring and control.
+Following lists the specific metrics that are monitored on Arm Juno r2:
+
+* Power telemetry:
+
+The sysfs interface provides a file for each of the following
+Advanced Peripheral Bus (APB) energy meter registers:
+
+    * SYS_POW_SYS : /sys/class/hwmon/hwmon0/power1_input
+    * SYS_POW_A72 : /sys/class/hwmon/hwmon0/power2_input
+    * SYS_POW_A53 : /sys/class/hwmon/hwmon0/power3_input
+    * SYS_POW_GPU : /sys/class/hwmon/hwmon0/power4_input
+
+Instantaneous power is recorded in bits 0-23 and are reported in microwatts by the
+sysfs interface. Variorum reports power in milliwatts. All registers are updated every
+100 microseconds.
+
+* Thermal telemetry:
+
+The sysfs interface provides a file for thermal telemetry for each system component
+through the GetSensorValue command in CSS System Control and Power Interface (SCPI).
+
+    * SoC temperature :       /sys/class/hwmon/hwmon0/temp1_input
+    * Big temperature :       /sys/class/hwmon/hwmon0/temp2_input
+    * Little temperature :    /sys/class/hwmon/hwmon0/temp3_input
+    * GPU temperature :       /sys/class/hwmon/hwmon0/temp4_input
+
+Instantaneous temperatures are reported in degree Celsius.
+
+* Clocks telemetry:
+
+Clocks are collected by the sysfs interface using the GetClockValue command in SCPI.
+A policy subdirectory is provided for big and little clusters each.
+
+    * Big clocks :    /sys/devices/system/cpu/cpufreq/policy0/scaling_cur_freq
+    * Little clocks : /sys/devices/system/cpu/cpufreq/policy1/scaling_cur_freq
+
+Frequencies are reported by the sysfs interface in Hz. Variorum reports the
+clocks in MHz.
```

Comment:
Same question here, should variorum just report out the same units coming from the interface?

Reply:
Actually, that's a typo -- it needs to be KHz instead of Hz. I introduced the conversion to MHz so that the frequency values have the same metric as the input to 'set_socket_frequency' which is in MHz. Should both be in KHz?

Reply:
Add a note to source and doc: Variorum reports clocks in MHz in consistent with clocks reported by Variorum for other architectures.